setup.cfg

```diff
@@ -1,2 +1,5 @@
 [aliases]
 testing = develop easy_install eduid_userdb[testing]
+
+[isort]
+sections = FUTURE,STDLIB,THIRDPARTY,FIRSTPARTY
```

Comment:
I believe these would be the defaults?

I think we should either try to have the same isort paragraph in all projects setup.cfg, or you should not even include it here where I guess no EDUID section is needed.